### PR TITLE
GG-151: Winner page prettier

### PR DESCRIPTION
**Pull request by whole01**
*  dark design 

Winner.component:
* show winner name in title
* show sorted leaderboard
* destroy game on destruction of winner page
* relocated winner.component files

### CODE REVIEW

File: src/app/components/pages/winner/winner.component.ts [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+  constructor(private readonly router: Router, activatedRoute: ActivatedRoute, private gameService: GameService) {
+    this.gameId = activatedRoute.snapshot.paramMap.get('gameId');
+    this.gameService.getGameFromServer(this.gameId).subscribe(game => {
+      this.sortLeaderboardAccordingToGameMode(game);
+    });
+  }
+
+
+  private sortLeaderboardAccordingToGameMode(value: Game) {
+    if (value.gameMode.name === '301' || value.gameMode.name === '501' ) {
```
Comment (mohjohfox) [inline src/app/components/pages/winner/winner.component.ts:26]: Nice one here :)

File: src/app/components/pages/winner/winner.component.ts [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+      this.leaderboard.playersAsArray = value.players.sort((p1, p2) => p1.points - p2.points);
```
Comment (mohjohfox) [inline src/app/components/pages/winner/winner.component.ts:27]: Nice one here :)

**Review by mohjohfox — APPROVED** [review]

File: src/styles.css [hunk 3]
```diff
@@ -1,4 +1,15 @@
 /* You can add global styles to this file, and also import other style files */
+body.dark-mode {
+  background-color: #2d3436;
+  color: #dfe6e9;
+}
+
+body.light-mode {
```
[the remark this answers is not in the document]
Comment (whole01) [reply src/styles.css:7]: I think we don't need it. I will remove it in the next story.

[the remark this answers is not in the document]
Comment (whole01) [reply src/styles.css:7]: I think we don't need it. I will remove it with the next story.

**Pull request merged**